rpc/handlers.go

```diff
@@ -110,17 +111,41 @@ func (h *Handler) BlockWithTxHashes(id BlockID) (*BlockWithTxHashes, *jsonrpc.Er
 		txnHashes[index] = txn.Hash()
 	}
 
+	l1H, err := h.l1Head()
+	if err != nil {
+		return nil, ErrInternal
```

Comment:
could be nice to use json.Err and pass err.Error() as data

Reply:
Ok will update

Reply:
fixed